task_2

```diff
@@ -0,0 +1,15 @@
+import functools
+import tracemalloc
+
+
+def memory(func):
+
+    @functools.wraps(func)
+    def wraps(*args, **kwargs):
+        tracemalloc.start()
+        func(*args, **kwargs)
+        snapshot = tracemalloc.take_snapshot()
+        tracemalloc.stop()
+        stats = snapshot.statistics('Traceback')[0]
```

Comment:
stats = snapshot.statistics('Traceback')[0] 
here better to unpack values
min_memory, max_memory = snapshot.statistics('Traceback')